modules/dag-shared/src/main/scala/org/tessellation/dag/snapshot/GlobalSnapshot.scala

```diff
@@ -51,12 +50,6 @@ case class GlobalSnapshot(
 
 object GlobalSnapshot {
 
-  implicit def encoder[G[_]: KryoSerializer]: EntityEncoder[G, GlobalSnapshot] =
-    BinaryCodec.encoder[G, GlobalSnapshot]
-
-  implicit def decoder[G[_]: Concurrent: KryoSerializer]: EntityDecoder[G, GlobalSnapshot] =
-    BinaryCodec.decoder[G, GlobalSnapshot]
-
```

Comment:
Don't we need binary codecs for returning GlobalSnapshot from L0 public endpoints? 🤔

Reply:
We need, but we don't need a special `EntityDecoder` instance for it. It's enough to import this in the server
```
import org.tessellation.ext.codecs.BinaryCodec.encoder
```
and this in the client
```
import org.tessellation.ext.codecs.BinaryCodec.decoder
```